bindings/rust/Cargo.toml

```diff
@@ -10,7 +10,8 @@ members = [
 # generate/main.rs
 exclude = [
     "generate",
-    "bench"
+    "bench",
+    "examples",
```

Comment:
I'd prefer we put examples in their own workspace so we don't get implicit dependencies/features enabled by our internal or other examples dependencies